tests/full_node_test.cpp

```diff
@@ -964,12 +964,12 @@ TEST_F(FullNodeTest, sync_two_nodes1) {
   auto num_vertices1 = nodes[0]->getDagManager()->getNumVerticesInDag();
   auto num_vertices2 = nodes[1]->getDagManager()->getNumVerticesInDag();
   for (unsigned i = 0; i < SYNC_TIMEOUT; i++) {
-    if (num_vertices1.first > 3 && num_vertices2.first > 3 && num_vertices1 == num_vertices2) break;
+    if (num_vertices1.first > 1 && num_vertices2.first > 1 && num_vertices1 == num_vertices2) break;
```

Comment:
We limit the max trxs in one DAG block that is 250, 1000 trxs should be at least in 4 DAG blocks. So the original test is "> 3"

Reply:
Fixed